ansible/roles/accumulo/templates/accumulo-cluster-systemd

```diff
@@ -75,6 +75,7 @@ function verify_config {
     fi
   fi
 
+{% if accumulo_version is version('2.1.0','<') %}
```

Comment:
I'm not a jinja expert, and this would totally be a nit, but you can specify a dash after the begin and or before the close. That will remove whitespace. A pattern I've seen before is something like:
```
{%- some jinja conditional %}
stuff
{% end -%}
```
The only difference this would make is how clean the file looks after it's been run through the template engine.

Reply:
I tried playing around with this feature locally and seeing how it rendered.  I didn't use it, but as a result I moved a few empty lines there were outside the template to inside of it.